### ISSUE TITLE
Header logo lacks accessible text and improper use of Title attribute

### ISSUE THREAD

**Issue by jhung** [pressbooks/pressbooks-book#170]
### Prerequisites
* [x] Can you reproduce the behavior you wish to address with [all plugins except Pressbooks deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/), the latest version of [Pressbooks Book](http://github.com/pressbooks/pressbooks-book/releases/latest/), the latest version of [this book theme](http://github.com/pressbooks/pressbooks-book/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing the issue while using this book theme?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks-book) to see if your bug or enhancement is already listed?

### Description

The header logo and the associated anchor element on cover page, and content pages is not screen reader accessible - it lacks accessible text and does not use the `title` attribute properly.

1. The logo link should at minimum have the `title` attribute should be removed and replaced by an `aria-label` attribute instead.

Aside: Title should not be used as a primary method of conveying meaning. See these articles on [Yoast](https://yoast.com/dev-blog/dont-rely-title-attribute/) and [WebAim](https://webaim.org/articles/gonewild/#title).

2. The logo SVG should have `aria-role=image`.

Ideally the word “Pressbooks” in the SVG were text in the document so it’s screen reader accessible, and be given an `<h1>` for better landmarking
